airflow/contrib/hooks/gcs_hook.py

```diff
@@ -269,3 +269,32 @@ def list(self, bucket, versions=None, maxResults=None, prefix=None, delimiter=No
                 # empty next page token
                 break
         return ids
+
+    def get_size(self, bucket, object):
+        """
+        Gets the size of a file in Google Cloud Storage.
+        :param bucket: The Google cloud storage bucket where the object is.
+        :type bucket: string
+        :param object: The name of the object to check in the Google cloud
+            storage bucket.
+        :type object: string
+        """
+        service = self.get_conn()
+        try:
+            response = service.objects().get(
+                bucket=bucket,
+                object=object
+            ).execute()
+
+            if 'name' in response and response['name'][-1] != '/':
+                # Remove Directories & Just check size of files
+                if 'size' in response:
```

Comment:
This method seems to return nothing if size is not in response. What's the expected behavior here?